data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,145 @@ mission "Strider notification"
+
+
+
+mission "Station Exploration 2"
+	priority
+	name "Bring students home to <planet>"
+	description "Bring a group of Hai and human classmates back home to <destination>."
+	passengers 5
+	source "Darkrest"
+	destination "Greenwater"
+	to offer
+		has "Station Exploration 1: done"
+		not "Station Exploration Abandon Ship: declined"
+	on offer
+		conversation
+			`The spaceport is bathed in red light, and the booming voice continues to play. Your red-faced passengers rush back onto your ship shouting at each other in a mixture of languages, but once they're through the airlock you take off before something worse happens.`
+				launch
+	on visit
+		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		conversation
+			`Upon landing, you are accosted by the local police on <planet>.`
+			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
+			choice
+				`	"Am I in trouble?"`
+					goto normal
+				`	"I'm innocent!"`
+					goto normal
+				`	(Run away.)`
+			`	Almost as soon as you turn around to run you feel a sharp pain in your side as your consciousness fades away.`
+			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
+			label normal
+			`	"We just want to let you know what happened," the officer says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
+			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. They're not in trouble, as there's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
+			choice
+				`	"So, am I in trouble or not?"`
+					goto trouble
+				`	"Are my passengers going to be okay?"`
+			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`
```

Comment:
```suggestion
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though.`
```

Reply:
The next line starts with a quotation, though, and I imagine it's the same officer speaking both lines.

Reply:
Oh I see what you mean.